.env

```diff
@@ -1,12 +1,12 @@
 # Use .env.local to change these variables
 # DO NOT EDIT THIS FILE WITH SENSITIVE DATA
 
-MONGODB_URL=#your mongodb URL here
+MONGODB_URL=mongodb://localhost:27017
 MONGODB_DB_NAME=chat-ui
 MONGODB_DIRECT_CONNECTION=false
 
 COOKIE_NAME=hf-chat
-HF_ACCESS_TOKEN=#hf_<token> from from https://huggingface.co/settings/token
+HF_ACCESS_TOKEN=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
```

Comment:
Looks like you leaked your HF token here. Don't put your secrets in the `.env`, it should go in `.env.local` and I'd recommend deleting this secret and using a new one since this one leaked.